src/utils/condition.js

```diff
@@ -3,21 +3,18 @@
 const crypto = require('crypto')
 const base64url = require('./base64url')
 
-function toConditionUri (conditionPreimage) {
+function toCondition (conditionPreimage) {
```

Comment:
Can we move these functions into the crypto helper?

Also, I think it should export a `generateCondition(sharedSecret, packet)` function instead of `toCondition` from the fulfillment.